cla_backend/libs/eligibility_calculator/calculator.py

```diff
@@ -421,33 +429,33 @@ def _is_data_complete_enough_to_call_cfe(case_data):
         if EligibilityChecker._under_18_passported(case_data):
             # no more info needed
             return True
-        if (
-            not hasattr(case_data.facts, "dependants_young")
-            and not (hasattr(case_data.facts, "on_passported_benefits") and case_data.facts.on_passported_benefits)
+        if not hasattr(case_data.facts, "dependants_young") and not (
+            hasattr(case_data.facts, "on_passported_benefits") and case_data.facts.on_passported_benefits
         ):
             # the gross income threshold may increase, depending on the number of child dependants,
             # so we can't do gross income section - even if we tell CFE it is incomplete, if the gross income
             # is over the threshold it will give 'ineligible'.
             # Therefore to run CFE, we need to know dependants_young, or if they are passported (because the
             # gross income check doesn't come into play)
-            logger.info("Eligibility check: CFE can't be called because dependants_young not known (and not passported)")
+            logger.info(
+                "Eligibility check: CFE can't be called because dependants_young not known (and not passported)"
+            )
             return False
         return True
 
     @staticmethod
     def _translate_case(case_data, submission_date=None):
-        '''Translates CLA's CaseData to CFE-Civil request JSON'''
+        """Translates CLA's CaseData to CFE-Civil request JSON"""
         if not submission_date:
             submission_date = datetime.date.today()
-        # produce the simplest possible plain request to CFE to prove the route
+        # default DOB to 'nothing special' 40 years old rules-wise
+        default_dob = str(datetime.date(submission_date.year - 40, submission_date.month, submission_date.day))
```

Comment:
Needs something like this to cope with leap years:
```suggestion
        default_dob = str(submission_date - relativedelta(years=40))
```

Reply:
I think this line of code can go. Didn't realise it was still present